aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -16,6 +16,7 @@ feature_flags:
     - resource_groups
     - multisig_accounts
     - delegation_pools
+    - ed25519_pk_validate_no_abort_on_wrong_length  
```

Comment:
Can we not enable this feature for either devnet genesis or this (which will be for testnet) before the actual feature is complete and proven to work? Adding the feature flag definition here is okay

Reply:
So you mean remove this line because this enables it on both devnet & testnet?

Reply:
Kevin confirmed to actually leave it enabled here, which only affects devnet. Testnet flags are done differently.

But, that being said, the flag name is no longer consistent with your recent rename, AFAICT.

Reply:
i.e., with `Ed25519PubkeyValidateReturnFalseWrongLength`